setup.py

```diff
@@ -61,9 +61,11 @@ def embed_commit_hash():
 
 VLLM_TARGET_DEVICE = envs.VLLM_TARGET_DEVICE
 
-# vLLM only supports Linux platform
-assert sys.platform.startswith(
-    "linux"), "vLLM only supports Linux platform (including WSL)."
+if not sys.platform.startswith("linux"):
```

Comment:
This change actually makes it possible to install the published `tar.gz` on mac, by setting `VLLM_TARGET_DEVICE` to `"empty"`. Also logging a warning about vLLM not actually being able to run.